judge/forms.py

```diff
@@ -20,6 +20,21 @@
 from judge.widgets import HeavyPreviewPageDownWidget, MathJaxPagedownWidget, PagedownWidget, Select2MultipleWidget, \
     Select2Widget
 
+totp_or_scratch_code_validators = {
```

Comment:
```suggestion
two_factor_validators_by_length = {
```

just to make clear what the key in this dict is